### DISCUSSION TITLE
Change load in multiple step engine file

### DISCUSSION THREAD

**Discussion by halfwitbm** [OpenRadioss/OpenRadioss#3265] [Q&A] [answered]
Dear experts,

Whether we can change load data in multiple steps in the engine files? For example: after finishing first running step (for example: 0~5s) , change displacement for some rigid bodies or soft bodies in engine input and then continue another  run step (for example 5~10s).
If you have some experience on this side, please give me some directions. Thank you in advance.

Regards,

Bin

**Comment by PaulAltair** [comment]
There are no steps as such, but, you may be able to achieve what you need to by modifying the applied function in a given engine. If you know the timings in advance you may just specify load vs time accordingly, if you need to start/stop based on some model condition, you can use /STOP/LSENSOR and /FUNCT modification in engine together?

**Reply by halfwitbm** [reply]
Hi Paul,

Thank you very much for your answer. Unfortunately, we don't know the curve of load vs time. May I simplify my story like this: one beam which has length l, one side is fixed, on other side, we can give it a displacement dx, dy or dz, after some time we change its displacement (on movable side) to dx1, dy1 or dz1.  after the simulation at the first stage, we output the restart file, for the second stage simulation, whether we can impose new load information by modify engine input file or modify something in restart file?  do you know what is the best way to accomplish it? (To regenerate starter file is not a efficient way). Hope I can get suggestion from you. Thank you very much.

Regards

Bin

**Reply by MarianBulla** [reply]
Hi @halfwitbm,

and happy new year 2025 !

regarding your question, how about using multiple ENGINE files ?

You can modify the functions and some other properties in the next/following runs within the engine file.
For more details, please have a look at the manual: https://help.altair.com/hwsolvers/rad/topics/solvers/rad/engine_input_r.htm

Hope it helps you?

Thanks and best regards,
Marian

**Reply by halfwitbm** [reply, marked as answer]
Hi Marian,

Happy new year 2025. Your information is very helpful. Thank you very much. 

Regards,

Bin
